packages/api-server/src/index.ts

```diff
@@ -15,9 +15,16 @@ type LaunchOpts = {
   googleOauthClientSecret?: string;
 };
 
+export type ServerClose = ReturnType<
+  typeof express["application"]["listen"]
+>["close"];
+export type Server = {
+  close: ServerClose;
```

Comment:
rename to `onClose`?

Reply:
this is mimicking the express.js server object